db/migrate/20171018081206_showback_fix_and_rename.rb

```diff
@@ -0,0 +1,51 @@
+class ShowbackFixAndRename < ActiveRecord::Migration[5.0]
+  def self.up
+    rename_table :showback_events,      :data_rollups
+    rename_table :showback_charges,     :data_views
+    rename_table :showback_price_plans, :price_plans
+    rename_table :showback_pools,       :envelopes
+    rename_table :showback_rates,       :rates
+    rename_table :showback_tiers,       :tiers
+    rename_table :showback_usage_types, :input_measures
+
+    rename_column :data_views, :showback_event_id,      :data_rollup_id
+    rename_column :data_views, :showback_pool_id,       :envelope_id
+    rename_column :rates,      :showback_price_plan_id, :price_plan_id
+    rename_column :tiers,      :showback_rate_id,       :rate_id
+
+    rename_column :rates,      :category,      :entity
+    rename_column :rates,      :dimension,     :field
+    rename_column :rates,      :measure,       :group
+    rename_column :rates,      :step_variable, :tier_input_variable
+    rename_column :rates,      :date,          :start_date
+
+    rename_column :input_measures, :category,   :entity
+    rename_column :input_measures, :dimensions, :fields
+    rename_column :input_measures, :measure,    :group
```

Comment:
Please add column :start_time and end_time to data_views, so we can store the corresponding information from the event.
A column with context too jsonb